src/rust-crypto/rust-crypto.ts

```diff
@@ -654,9 +654,31 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
         if (userIdentity === undefined) {
             return new UserVerificationStatus(false, false, false);
         }
+
         const verified = userIdentity.isVerified();
+        const wasVerified = userIdentity.wasPreviouslyVerified();
```

Comment:
There is another PR doing that https://github.com/matrix-org/matrix-js-sdk/pull/4411 . Ok for me to close it in favor of this one